packages/docs/docs/referrals/transmition-and-tracking.md

```diff
@@ -0,0 +1,87 @@
+# Referral Transmission & Tracking
+
+## Transmitting Referrals
+
+Referrals need to be securely transmitted to receiving providers. This can be done using various methods depending on the technical capabilities of the receiving system.
+
+The [`Communication`](/docs/api/fhir/resources/communication) resource is used to represent the transmission event, including details about the sender, recipient, and payload. For recipients without a FHIR API, a [`Bot`](/docs/bots/) can be used to generate and send a PDF of the referral information.
+
+## Tracking Referral Status
```

Comment:
You need a section on common data needed in referrals.  Also, no mention of CCD-A?  The primary use case of Transitions of care is referral /docs/integration/c-cda#transitions-of-care